PersonApi/V1/Infrastructure/StringExtensions.cs

```diff
@@ -0,0 +1,22 @@
+namespace Hackney.Core
+{
+    // TODO - move to common nuget package
+
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Converts the string to camel case (i.e. the first character is lowercase)
+        /// </summary>
+        /// <param name="str">The string to change</param>
+        /// <returns>A copied of the string with the first chacater in lowercase. A null or empty string returns what was supplied.
+        /// </returns>
+        public static string ToCamelCase(this string str)
```

Comment:
The naming here - I think this only turns the first letter of the word to a lowercase. So, if the word is SomeWord, this makes it someWord (which is correct). But if it's Someword, this returns someword (which is incorrect). or am I wrong?

Reply:
I think you're wrong. Strictly I guess it should be `ToLowerCamelCase`.
And this is exactly what the json seriliazers also do (which is why I need it).

Reply:
All I am saying is - this will not turn a fully lowercase word into camelcase. It will turn a Camelcase to LowerCamelCase. :) thiswordrighthere won't turn into thisWordRightHere (which is the lowerCamelCase). Right?

Reply:
No it wouldn't do that.
But nothing could without a language model and intelligent language processing. If an entire text string is lower case how would it know which characters to make upper case.